scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -164,6 +172,7 @@ package object dialects {
     def allowAndTypes = Scala211.allowAndTypes
     def allowOrTypes = Scala211.allowOrTypes
     def allowTraitParameters = Scala211.allowTraitParameters
+    def allowLiteralTypes: Boolean = false // Scheduled to be included in 2.12.2
```

Comment:
If I understand correctly, after 2.12.2 is released, we should flip the flag, right? If yes, let's create a ticket for that.